Meadowlark-js/backends/meadowlark-mongodb-backend/docker/debezium-mongodb.json.example

```diff
@@ -1,12 +1,26 @@
 {
-  "name": "connector",
+  "name": "DebeziumConnector",
   "config": {
       "connector.class": "io.debezium.connector.mongodb.MongoDbConnector",
       "tasks.max": "1",
-      "mongodb.name": "meadowlark",
+      "mongodb.name": "edfi",
       "mongodb.user": "<admin-username-goes-here>",
       "mongodb.password": "<admin-password-goes-here>",
       "mongodb.hosts": "rs0/mongo1:27017,mongo2:27018,mongo3:27019",
-      "database.history.kafka.bootstrap.servers": "kafka:9092"
+      "database.history.kafka.bootstrap.servers": "kafka:9092",
+
+      "key.converter": "org.apache.kafka.connect.json.JsonConverter",
+      "key.converter.schemas.enable": "false",
+      "value.converter": "org.apache.kafka.connect.json.JsonConverter",
+      "value.converter.schemas.enable": "false",
+
+      "predicates": "isTombstone",
+      "predicates.isTombstone.type": "org.apache.kafka.connect.transforms.predicates.RecordIsTombstone",
+
+      "transforms": "dropTombstone,extractDebeziumAfter",
+      "transforms.dropTombstone.type": "org.apache.kafka.connect.transforms.Filter",
+      "transforms.dropTombstone.predicate": "isTombstone",
```

Comment:
Filter out the separate tombstone messages. We are actually want them, but are going to create them by side-effect from the delete messages themselves. Delete messages have no "after" field.